diff --git a/build.gradle b/build.gradle
index 7686167bd2..545d93c36e 100644
--- a/build.gradle
+++ b/build.gradle
@@ -35,16 +35,7 @@
 //----------------------END "DO NOT MODIFY" SECTION-------------------------------
 
 def getGitHash = {
-    def stdout = new ByteArrayOutputStream()
-    exec {
-        if (System.getProperty('os.name').toLowerCase(Locale.ROOT).contains('windows')) {
-            commandLine 'cmd', '/c', 'git', 'rev-parse', '--short', 'HEAD'
-        } else {
-            commandLine 'git', 'rev-parse', '--short', 'HEAD'
-        }
-        standardOutput = stdout
-    }
-    return stdout.toString().trim()
+    return System.env.GSL_COMMIT
 }
 
 repositories {
