diff --git a/src/filter.c b/src/filter.c
index f3e41ff..8ca5e15 100644
--- a/src/filter.c
+++ b/src/filter.c
@@ -33,6 +33,7 @@
 #include "epson-protocol.h"
 #include "epson-escpr-api.h"
 #include "epson-escpr-mem.h"
+#include "epson-escpr-services.h"
 
 #include "err.h"
 #include "mem.h"
diff --git a/src/mem.c b/src/mem.c
index f53f2bc..d6c945a 100644
--- a/src/mem.c
+++ b/src/mem.c
@@ -23,6 +23,7 @@
 
 #include <stdlib.h>
 #include "mem.h"
+#include "err.h"
 
 void *
 mem_malloc (unsigned int size, bool_t crit)
